### PR TITLE
extra ";" generation has been taken care for Bang command

### LINKED ISSUE #851: [BUG]: `!set ...` commands break Scala functional tests
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Category of Bug / Issue

Other

### Current Behavior

```
 === Unexpected output (expected vs actual) ===
!-- snowsql command:!'set exit_on_error = true';   --!set exit_on_error = true;;
```
image

### Expected Behavior

There must be no errors

### Steps To Reproduce

run `SnowflakeAcceptanceSuite`

### Relevant log output or Exception details

_No response_

### Sample Query

_No response_

### Operating System

macOS

### Version

latest via Databricks CLI

### PR DESCRIPTION
**Pull request by sriram251-code**
fixes  #851

### CODE REVIEW

**Review by jimidle — APPROVED** [review]
lgtm

File: tests/resources/functional/snowflake/test_command/test_command_1.sql [hunk 1]
```diff
@@ -3,4 +3,4 @@
 !set exit_on_error = true;
 
 -- databricks sql:
--- snowsql command:!'set exit_on_error = true';
+--!'set exit_on_error = true';
```
[the remark this answers is not in the document]
Comment (sriram251-code) [reply tests/resources/functional/snowflake/test_command/test_command_1.sql]: done

File: core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala [hunk 2]
```diff
@@ -36,10 +37,26 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
...
+    //  TODO We should always generate an unresolved node, our plan should never be null
     case null => "" // don't fail transpilation if the plan is null
     case x => throw unknown(x)
   }
 
+  private def batch(ctx: GeneratorContext, b: Batch): String = {
+    val seqSql = b.children
+      .map {
+        case ir.UnresolvedCommand(text) =>
+          "--" + text.stripSuffix(";")
```
[the remark this answers is not in the document]
Comment (sriram251-code) [reply core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala]: done

**Comment by sriram251-code** [comment]
> image code is good, but you've added +2 failures. fix those

Its shows the old results the new results are as follows. I am not sure why its showing the old result.
Screenshot 2024-08-28 at 10 29 41 PM https://github.com/databrickslabs/remorph/runs/29379915616

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**